config.js

```diff
@@ -29,3 +29,8 @@ cfg.mongoUrlTest = process.env.MONGO_URL_TEST || defaultMongoUrlTest;
 
 // Export configuration object
 module.exports = cfg;
+
+
+if (!cfg.twilioAccountSid || !cfg.twilioAuthToken) {
```

Comment:
This looks good. I'm wondering if you would consider to include `dotenv-safe` package that @joliveros mentioned?